README.md

```diff
@@ -3,6 +3,7 @@
 ![](https://travis-ci.com/UCLA-CS130/Mr.-Robot-et-al..svg?token=yz9yBJgd4Sxya2e1weD1&branch=master)
 
 :zap: Lightning is a modern, robust web server written in C++.
+This repository continues to develop atop its [original incarnation](https://github.com/UCLA-CS130/Mr.-Robot-et-al.).
```

Comment:
This should be slightly rephrased for clarity: "...continues to develop upon the original version of Lightning"